server/graphql/common/expenses.ts

```diff
@@ -1612,21 +1612,23 @@ const generateInsufficientBalanceErrorMessage = ({
   balance,
   currency,
   expenseAmount,
+  expenseCurrency,
+  rate,
   isSameCurrency,
   fees = 0,
   feesName = '',
 }) => {
   let msg = `Collective does not have enough funds to ${object}.`;
   msg += ` Current balance: ${formatCurrency(balance, currency)}`;
-  msg += `, Expense amount: ${formatCurrency(expenseAmount, currency)}`;
+  msg += `, Expense amount: ${formatCurrency(expenseAmount, expenseCurrency)}`;
   if (fees) {
-    msg += `, Estimated ${feesName} fees: ${formatCurrency(fees, currency)}`;
+    msg += `, Estimated ${feesName} fees: ${formatCurrency(fees, expenseCurrency)}`;
   }
 
   if (!isSameCurrency) {
-    msg += `. For expenses submitted in a different currency than the collective, an error margin of 20% is applied. The maximum amount that can be paid is ${formatCurrency(
-      Math.round(balance / 1.2),
-      currency,
+    msg += `. For expenses submitted in a different currency than the collective, an error margin of 2σ to the latest rate is applied. The maximum amount that can be paid is ${formatCurrency(
```

Comment:
I'm not sure all users will be comfortable with the concept of standard deviation. I'd argue for simplification:

```suggestion
    msg += `. For expenses submitted in a different currency than the collective, an error margin is applied to accommodate for fluctuations. The maximum amount that can be paid is ${formatCurrency(
```